### ISSUE TITLE
Command-line interpreters for lexer and parser

### ISSUE THREAD

**Issue by adamjstewart** [adamjstewart/prolog#5]
For demonstration purposes, it would be nice to have command-line interpreters for the lexer and parser so we can interactively demo them.

**Comment by ZaidQureshi** [comment]
Yes, be my guest to make it. It should be very similar to the already existing interpreter, except for the fact that instead of calling eval_dec you print the input. Or you can even print this in the current interpretor.
